pkg/servicemesh/federation/common/util.go

```diff
@@ -15,17 +15,25 @@
 package common
 
 import (
+	"crypto/sha256"
+	"encoding/hex"
 	"fmt"
 
 	"github.com/mitchellh/hashstructure/v2"
 	discoveryv1 "k8s.io/api/discovery/v1"
 	kubelabels "k8s.io/apimachinery/pkg/labels"
 	v1 "maistra.io/api/federation/v1"
 
+	"istio.io/istio/pkg/config/labels"
 	"istio.io/istio/pkg/kube"
 	"istio.io/istio/pkg/spiffe"
 )
 
+var (
+	maxNameLength = 63
```

Comment:
You can remove this constant and use `labels.DNS1123LabelMaxLength` instead.

Reply:
There is this [comment](https://github.com/maistra/istio/blob/maistra-2.3/pkg/config/labels/instance.go#L27) for the usage of `DNS1123LabelMaxLength` variable. Does it mean for testing in the condition blocks or for unit tests ?

Reply:
I think they just meant that the const isn't used outside the file (except in tests). But we can definitely use it instead of creating our own copy.